tasks/typescript/easy/index.ts

```diff
@@ -1,9 +1,9 @@
 // TypeScript - Easy
 
 function greet(name: string): string {
-    // TODO: Add return statement
+  return name;
```

Comment:
This should return a greeting message "Hello, John Doe!" as it is requested in the issue description. Right now it returns the name John Doe.